### PR TITLE
Week3 homework-Lambda function, cloudwatch

### PR DESCRIPTION
**Pull request by rajanibusani**

### CODE REVIEW

File: week3/materials/homework-cron-lambda/assignments.md [hunk 1]
```diff
@@ -8,12 +8,12 @@ doc: `https://docs.aws.amazon.com/cli/latest/reference/lambda/list-functions.htm
...
-doc: `?`
+doc: `https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-lambda-function.html`
 
 **Assignment 3:**
-rubbish string: `?`
+rubbish string: `Ifmmp-!IZG\`
 
 **Assignment 4:**
-total count: `?`
+total count: `1`
```
Comment (paulinadubas) [inline week3/materials/homework-cron-lambda/assignments.md:19]: 👍🏻

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,4 @@
+**/.DS_Store
+**/node_modules
+**/build
+**/package-lock.json
```
Comment (paulinadubas) [inline .gitignore]: New line at the end of file is a good practice :)

**Review by paulinadubas — APPROVED** [review]
It looks correct to me :)

File: .gitignore (hunk 2 above)
Comment (rajanibusani) [reply .gitignore]: Ok...I will, Thank You @paulinadubas for your time

**Pull request merged**